spacefi.io/chains.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "../chains.schema.json",
+  "name": "Citizen Cosmos",
```

Comment:
Can you just check this file, I assume this was missed. Want to make sure your address etc is correct too.